Aptoide/aptoide-client-v8
# [ASV-218] Added a fragment with a webView to act as a generic display…

**What does this PR do?**

This PR aims to add a fragment to show the terms and conditions. This is using a webview to display the data from the web.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] GenericWebviewFragment.java
- [ ] Settings.java

**How should this be manually tested?**

Go to settings and use the terms and conditions preference button at the bottom. It should take you places now;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-218](<https://aptoide.atlassian.net/browse/ASV-218>)

**Code Review Checklist**
